src/views/subscribe/subscribe.js

```diff
@@ -0,0 +1,235 @@
+      } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
+        errors.email = 'Invalid email address.';
+      }
+
+      return errors;
+    },
+    onSubmit: (values, actions) => {
+      sendData(values, actions)
+    },
+  });
+  
+  const { handleChange, values, handleBlur, handleSubmit, touched, errors } = formik;
+  
+  return (
+    <>
+      <Container sx={{ p: 4 }}>
+        <div className="row subscribe-class">
+          <div className="col-md-6 col-sm-12">
+            <div className="heading-class">
+              <span>
+                Get all the OpenEBS updates & information straight to your inbox 📨
+            </span>
+            </div>
+            <p>
+              <span>
+                👉
+                <span className="points-class">
+                  Monthly OpenEBS Newsletter
+                </span>
+              </span>
+            </p>
+            <p>
+              <span>
+                👉
+                <span className="points-class">
+                  Updates about latest releases
+                </span>
+              </span>
+            </p>
+            <p>
+              <span>
+                👉
+                <span className="points-class">
+                  Other OpenEBS Events
+                </span>
+              </span>
+            </p>
+
+          </div>
+          <div className="col-md-6 col-sm-12">
+            <label className="form-title-class">Fill the form</label>
+            <form onSubmit={handleSubmit}>
+              <div className="form-group">
+                <label>First Name</label>
+                <input type="text" className="form-control" name="firstName"
+                  onChange={handleChange}
+                  value={values.firstName} />
+              </div>
+              <div className="form-group">
+                <label>Last Name</label>
+                <input type="text" className="form-control" name="lastName"
+                  onChange={handleChange}
+                  value={values.lastName} />
+              </div>
+              <div className="form-group">
+                <label>Email*</label>
+                <input type="email" className="form-control" name="email"
+                  onChange={handleChange}
+                  onBlur={handleBlur}
+                  value={values.email} />
+                {errors && errors.email && touched.email ?
+                  <div className="text-danger">
+                    {errors.email}
+                  </div>
+                  : null}
+              </div>
+              <div className="btn-wrap">
+                <button type="submit" className="btn btn-primary">I"m In</button>
+              </div>
+            </form>
+          </div>
+        </div>
+        <div>
+          <ReactModal
+            className="modal-class"
+            isOpen={showModal}
+            contentLabel="modal"
+            onRequestClose={handleCloseModal}
+            style={{
```

Comment:
cannot we use `sx`?? or you can put inline styling in classes.

Reply:
We have to use props for ReactModal component so we can not give it those style as a class